apps/dashboard/src/components/modal/index.js

```diff
@@ -45,12 +54,30 @@ export const ModalHeader = styled.h2`
 
 export const ModalHeaderNote = styled.p`
 	color: var( --color-text-subtle );
-	font-size: 16px;
 	margin-bottom: 40px;
+
+	&&& {
```

Comment:
This is a trick to increase selector specificity: https://styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity
Necessary to override styles with higher precedence.